### ISSUE TITLE
PayPal Advanced - Default Setting

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#441]
I have a user that downloaded our release branch to try things out.  He says that upon updating the plugin, PayPal Advanced was activated by default.

This was especially confusing for him because he is using a currency code that is not supported by Advanced.  As such, when he would enter the Advanced settings panel he was given the message that it does not support his currency, and he was not able to disable it.

I told him to switch to USD, disable it, and then switch back.  This worked fine, but I could see this coming up for lots of different people.

Basically, we just need to make sure that Advanced is not activated by default.
